docs/fundamentals/code-analysis/configuration-options.md

```diff
@@ -51,22 +51,38 @@ Rule-specific options can be applied to a single rule, a set of rules, or all ru
 - [Rule severity level](#severity-level)
 - [Options specific to *code-quality* rules](code-quality-rule-options.md)
 
-To set one of these options for a single rule, use the following syntax.
```

Comment:
Moved this block under the `Severity level` section

Reply:
The reason I pulled it out of the severity level section is because it applies to the additional code-quality options too, e.g. `exclude_async_void_methods`.

Reply:
That has a different format though, and is specific to just the code quality rules. `dotnet_diagnostic.` and `dotnet_analyzer_diagnostic.` are reserved by compiler for severity options only. It would be ideal if same prefixes are shared by code quality options, but they are not as of today.

Reply:
Okay. We'll need to add a section back to the code quality rules options page then.

```
## Option scopes

Each refining option can be configured for all rules, for a category of rules (for example, Security or Design), or for a specific rule.

### All rules

The syntax for configuring an option for *all* rules is as follows:

|Syntax|Example|
|-|-|
| dotnet_code_quality.OptionName = OptionValue | `dotnet_code_quality.api_surface = public` |

### Category of rules

The syntax for configuring an option for a *category* of rules (such as Naming, Design, or Performance) is as follows:

|Syntax|Example|
|-|-|
| dotnet_code_quality.RuleCategory.OptionName = OptionValue | `dotnet_code_quality.Naming.api_surface = public` |

### Specific rule

The syntax for configuring an option for a *specific* rule is as follows:

|Syntax|Example|
|-|-|
| dotnet_code_quality.RuleId.OptionName = OptionValue | `dotnet_code_quality.CA1040.api_surface = public` |
```

Reply:
Ah, I did not realize that was removed. Let me do that.

Reply:
Done, can you please check again? Thanks.